modules/mongodb/charts.go

```diff
@@ -0,0 +1,490 @@
+		},
+	}
+	chartTcmalloc = module.Chart{
+		ID:    "tcmalloc",
+		Title: "Tcmalloc",
+		Units: "KiB",
+		Fam:   "tcmalloc",
+		Ctx:   "mongodb.tcmalloc",
+		Dims: module.Dims{
+			{ID: "tcmalloc.tcmalloc.pageheap_free_bytes", Name: "Pageheap free", Div: 1024},
+			{ID: "tcmalloc.tcmalloc.pageheap_unmapped_bytes", Name: "Pageheap unmapped ", Div: 1024},
+			{ID: "tcmalloc.tcmalloc.current_total_thread_cache_bytes", Name: "Total threaded cache", Div: 1024},
+			{ID: "tcmalloc.tcmalloc.total_free_bytes", Name: "Free", Div: 1024},
+			{ID: "tcmalloc.tcmalloc.pageheap_committed_bytes", Name: "Pageheap committed", Div: 1024},
+			{ID: "tcmalloc.tcmalloc.pageheap_total_commit_bytes", Name: "Pageheap total commit", Div: 1024},
+			{ID: "tcmalloc.tcmalloc.pageheap_total_decommit_bytes", Name: "Pageheap decommit", Div: 1024},
+			{ID: "tcmalloc.tcmalloc.pageheap_total_reserve_bytes", Name: "Pageheap reserve", Div: 1024},
+		},
+	}
+)
+
+var chartGlobalLockCurrentQueue = module.Chart{
+	ID:    "globalLockCurrentQueue",
+	Title: "Current Queue Clients",
+	Units: "clients",
+	Fam:   "clients",
+	Ctx:   "mongodb.currentQueue",
+	Type:  module.Stacked,
+	Dims: module.Dims{
+		{ID: "globalLock.currentQueue.readers", Name: "readers"},
+		{ID: "globalLock.currentQueue.writers", Name: "writers"},
+	},
+}
+
+var chartMetricsCommands = module.Chart{
+	ID:    "metricsCommand",
+	Title: "Command Metrics",
+	Units: "commands/s",
+	Fam:   "commands",
+	Ctx:   "mongodb.metricsCommand",
+	Dims: module.Dims{
+		{ID: "metrics.commands.eval.total", Name: "Eval", Algo: module.Incremental},
+		{ID: "metrics.commands.eval.failed", Name: "Eval Failed", Algo: module.Incremental},
+		{ID: "metrics.commands.delete.total", Name: "Delete", Algo: module.Incremental},
+		{ID: "metrics.commands.delete.failed", Name: "Delete Failed", Algo: module.Incremental},
+		{ID: "metrics.commands.count.failed", Name: "Count Failed", Algo: module.Incremental},
+		{ID: "metrics.commands.createIndexes", Name: "Create Indexes", Algo: module.Incremental},
+		{ID: "metrics.commands.findAndModify", Name: "Find And Modify", Algo: module.Incremental},
+		{ID: "metrics.commands.insert.failed", Name: "Insert Fail", Algo: module.Incremental},
+	},
+}
+
+var chartGlobalLocks = module.Chart{
+	ID:    "locks",
+	Title: "Locks",
+	Units: "locks/s",
+	Fam:   "locks",
+	Ctx:   "mongodb.locks",
+	Dims: module.Dims{
+		{ID: "locks.Global.acquireCount.r", Name: "Global Read Locks", Algo: module.Incremental},
+		{ID: "locks.Global.acquireCount.w", Name: "Global Write Locks", Algo: module.Incremental},
+		{ID: "locks.Database.acquireCount.r", Name: "Database Read Locks", Algo: module.Incremental},
+		{ID: "locks.Database.acquireCount.w", Name: "Database Write Locks", Algo: module.Incremental},
+		{ID: "locks.Collection.acquireCount.r", Name: "Collection Read Locks", Algo: module.Incremental},
+		{ID: "locks.Collection.acquireCount.w", Name: "Collection Write Locks", Algo: module.Incremental},
+	},
```

Comment:
```suggestion
var chartGlobalLocks = module.Chart{
	ID:    "locks",
	Title: "Acquired locks",
	Units: "locks/s",
	Fam:   "locks",
	Ctx:   "mongodb.locks",
	Dims: module.Dims{
		{ID: "locks.Global.acquireCount.r", Name: "global read", Algo: module.Incremental},
		{ID: "locks.Global.acquireCount.w", Name: "global write", Algo: module.Incremental},
		{ID: "locks.Database.acquireCount.r", Name: "database read", Algo: module.Incremental},
		{ID: "locks.Database.acquireCount.w", Name: "database write", Algo: module.Incremental},
		{ID: "locks.Collection.acquireCount.r", Name: "collection read", Algo: module.Incremental},
		{ID: "locks.Collection.acquireCount.w", Name: "collection write", Algo: module.Incremental},
	},
```

I think we can remove " Locks", it is clear from the context (the chart title).